src/resources/views/auth/account/sidemenu.blade.php

```diff
@@ -1,4 +1,10 @@
 <div class="box">
+    <div class="box-body box-profile">
+
```

Comment:
What's that extra line for?

Reply:
Following the overall style of the rest of the file.